### DISCUSSION TITLE
7681CDB7-66B2-740A-3845-1FC926CC162D

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#67] [General]
# 7681CDB7-66B2-740A-3845-1FC926CC162D

Minkowski 2-9 (also known as Minkowski's Butterfly, Twin Jet Nebula, the Wings of a Butterfly Nebula, or just Butterfly Nebula) is a planetary nebula located about 2,100 light-years away from Earth in the constellation Ophiuchus. The nebula was imaged by the Hubble Space Telescope in the 1990s

https://celestia.mobi/addon?item=7681CDB7-66B2-740A-3845-1FC926CC162D&lang=lt

**Comment by Einaras553** [comment]
This is too damn beutiful! Thus made my day. I hope you have a nice day.
